src/domain/events/utils.ts

```diff
@@ -29,18 +40,34 @@ export const getEventFilterVariables = (
   include: ['keywords,location'],
   text: getTextFromDict(query, 'text', undefined),
   inLanguage: getTextFromDict(query, 'inLanguage', undefined),
+  start: getDateString(query.date),
   ...options,
 });
 
+export const getDateString = (date?: string | string[]): string | null => {
+  return typeof date === 'string' && isValidDate(new Date(date))
+    ? new Date(date).toISOString()
+    : null;
+};
+
 export const getInitialValues = (
   query: NodeJS.Dict<string | string[]>
 ): EventSearchFormValues => {
   return {
     text: getTextFromDict(query, 'text') || '',
+    date: getInitialDate(query.date),
     inLanguage: queryParameterToArray(query.inLanguage as EVENT_LANGUAGES),
   };
 };
 
+export const getInitialDate = (date?: string | string[]): Date | null => {
```

Comment:
What is the reason to allow string array as date type?

Reply:
Same as above. query.date is a query parameter that can be string or string array.

query: NodeJS.Dict<string | string[]>